get-pmm.sh

```diff
@@ -1,22 +1,27 @@
 #!/usr/bin/env bash
 #
 # ###############################
-# Script to run PMM 2.
+# Script to run PMM 3.
 # If docker is not installed, this script will try to install it as root user.
+# if PMM 2 is running, it will be stopped and data will be migrated to PMM 3.
 #
 # Usage example:
-# curl -fsSL https://raw.githubusercontent.com/percona/pmm/main/get-pmm.sh -o get-pmm.sh; chmod +x get-pmm.sh; ./get-pmm.sh
+# curl -fsSL https://raw.githubusercontent.com/percona/pmm/main/get-pmm.sh -o get-pmm.sh; chmod +x get-pmm.sh; ./get-pmm.sh -b
 #
 #################################
 
 set -Eeuo pipefail
 trap cleanup SIGINT SIGTERM ERR EXIT
 
 # Set defaults.
+network_name=${NETWORK_NAME:-pmm-net}
 tag=${PMM_TAG:-3}
 repo=${PMM_REPO:-percona/pmm-server}
 port=${PMM_PORT:-443}
 container_name=${CONTAINER_NAME:-pmm-server}
+docker_socker_path=${DOCKER_SOCKER_PATH:-/var/run/docker.sock}
```

Comment:
Any reason it's `docker_socker_path`/`DOCKER_SOCKER_PATH` and not docker_socket_path/`DOCKER_SOCKET_PATH`?

Reply:
thanks, it's typo